internal/controllers/printresults/print_results.go

```diff
@@ -70,42 +78,42 @@ func (pr *PrintResults) SetAnalysis(entity *analysis.Analysis) {
 }
 
 func (pr *PrintResults) Print() (totalVulns int, err error) {
-	if err := pr.factoryPrintByType(); err != nil {
+	if err := pr.printByOutputType(); err != nil {
 		return 0, err
 	}
 
 	pr.checkIfExistVulnerabilityOrNoSec()
 	pr.verifyRepositoryAuthorizationToken()
 	pr.printResponseAnalysis()
 	pr.checkIfExistsErrorsInAnalysis()
-	if pr.configs.IsTimeout {
+	if pr.config.IsTimeout {
 		logger.LogWarnWithLevel(messages.MsgWarnTimeoutOccurs)
 	}
 
 	return pr.totalVulns, nil
 }
 
-func (pr *PrintResults) factoryPrintByType() error {
+func (pr *PrintResults) printByOutputType() error {
 	switch {
-	case pr.configs.PrintOutputType == outputtype.JSON:
-		return pr.runPrintResultsJSON()
-	case pr.configs.PrintOutputType == outputtype.SonarQube:
-		return pr.runPrintResultsSonarQube()
+	case pr.config.PrintOutputType == outputtype.JSON:
+		return pr.printResultsJSON()
+	case pr.config.PrintOutputType == outputtype.SonarQube:
+		return pr.printResultsSonarQube()
 	default:
-		return pr.runPrintResultsText()
+		return pr.printResultsText()
 	}
 }
 
-func (pr *PrintResults) runPrintResultsText() error {
-	fmt.Print("\n")
+func (pr *PrintResults) printResultsText() error {
+	fmt.Fprint(pr.writer, "\n")
 	pr.logSeparator(true)
 
-	pr.printLNF("HORUSEC ENDED THE ANALYSIS WITH STATUS OF \"%s\" AND WITH THE FOLLOWING RESULTS:", pr.analysis.Status)
+	pr.printlnf(`HORUSEC ENDED THE ANALYSIS WITH STATUS OF "%s" AND WITH THE FOLLOWING RESULTS:`, pr.analysis.Status)
 
 	pr.logSeparator(true)
 
-	pr.printLNF("Analysis StartedAt: %s", pr.analysis.CreatedAt.Format("2006-01-02 15:04:05"))
-	pr.printLNF("Analysis FinishedAt: %s", pr.analysis.FinishedAt.Format("2006-01-02 15:04:05"))
+	pr.printlnf("Analysis StartedAt: %s", pr.analysis.CreatedAt.Format("2006-01-02 15:04:05"))
```

Comment:
will we have the same problem as in #649 ?

Reply:
Here we are only printing the date. In this problem we was creating a file.

Reply:
LGTM